index.js

```diff
@@ -402,6 +402,23 @@
   //. ```
   S.K = def('K', [a, b], function(x, y) { return x; });
 
+  //. ### Function
+
+  //# flip :: (a -> b -> c) -> b -> a -> c
+  //.
+  //. Takes a binary function and two values and returns the result of
+  //. applying the function - with its argument order reversed - to the
+  //. values. `flip` may also be applied to functions with an arity
+  //. greater than two that have been appropriately curried.
```

Comment:
The second sentence would be clearer were we to refer to _function_ singular rather than _functions_ plural:

_`flip` may also be applied to a Ramda-style curried function with arity greater than two._